.github/workflows/build.yml

```
@@ -50,3 +50,27 @@ jobs:
           name: timecop_${{ env.PHP_API }}.so
           path: modules/timecop.so
           if-no-files-found: error
+  package:
+    name: Package for download
+    runs-on: ubuntu-20.04
```

Why specifically 20.04?

```suggestion
    runs-on: ubuntu-latest
```

---

So build doesn't break if the version of Ubuntu changes.